cxx-api-examples/punctuation-cxx-api.cc

```diff
@@ -0,0 +1,50 @@
+// cxx-api-examples/fire-red-asr-cxx-api.cc
```

Comment:
```suggestion
// cxx-api-examples/punctuation-cxx-api.cc
```

Please also change it to your info.